phoenicis-tools/src/main/java/org/phoenicis/tools/processes/ProcessUtils.java

```diff
@@ -0,0 +1,61 @@
+package org.phoenicis.tools.processes;
+
+import org.phoenicis.configuration.security.Safe;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+@Safe
+public class ProcessUtils {
+
+    /**
+     * Starts a new system process with the given command
+     *
+     * @param command     The command to be executed
+     * @param basePath    The location from where the command is executed
+     * @param environment A map of environemnt variables
+     * @param outputPath  An optional path where the output should be saved
+     * @return The created process
+     * @throws IOException thrown if the process couldn't be started
+     */
+    public PhoenicisProcess startProcess(List<String> command, String basePath, Map<String, String> environment, String outputPath) throws IOException {
```

Comment:
@plata @qparis should pass all parameters in a single call like I did here, or should be pass them using a builder API (i.e. using `withCommand`, `withBasePath` etc methods)?